src/com/theironyard/charlotte/Products.java

```diff
@@ -0,0 +1,70 @@
+package com.theironyard.charlotte;
+
+
+public class Products {
+    int id;
+    String productName;
+    String shortDescription;
+    double price;
+    int quantityAvailable;
+    String pictureURL;
+    boolean inCart=false;
+
+    public Products(int id, String productName, String shortDescription, double price, int quantityAvailable, String pictureURL,boolean inCart) {
+        this.id = id;
+        this.productName = productName;
+        this.shortDescription = shortDescription;
+        this.price = price;
+        this.quantityAvailable = quantityAvailable;
+        this.pictureURL = pictureURL;
+        this.inCart=inCart;
+    }
+
```

Comment:
Login doesn't work.  In create user you are never setting the username.  Why?  And then in your getUser method you are searching by username but they are inputting email address?  Fix this and resubmit.